src/main/java/com/andrewbrookins/idea/wrap/utils.kt

```diff
@@ -47,6 +53,18 @@ fun getWrapper(project: Project?, editor: Editor, fileIsPlaintext: Boolean): Cod
     return wrapper
 }
 
+fun shouldWrapLine(textData: TextData, isPlaintext: Boolean, fileExtension: String?): Boolean {
+    val isCommentLine = textData.lineData.indent.isNotBlank() && textData.lineData.rest.isNotBlank()
+    val plaintextWithoutSymbol = isPlaintext && textData.lineData.meaningfulSymbol.isBlank()
+    var isDocstring = false
+    if (!fileExtension.isNullOrEmpty() && (fileExtension == "py" || fileExtension == "kt")) {
+        val tripleDoubleQuote = "\"\"\""
+        val tripleSingleQuote = "'''"
+        val pattern = ("^\\s*(?:$tripleDoubleQuote|$tripleSingleQuote)\\s*$").toRegex()
```

Comment:
Seems like this doesn't work exactly right yet. Just trying to stop wrapping from wrapping """ and ''' in Python and Kotlin docstrings.